src/components/pages/Categories.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categories';
+
+const Categories = () => {
+  const status = useSelector((state) => state.checkStatus);
+  const dispatch = useDispatch();
 
-function Categories() {
   return (
     <>
       <div>
-        <button type="button">Check status</button>
+        <button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
+        <p>{status}</p>
```

Comment:
- [ ] {OPTIONAL] Kindly make sure when a user clicks on the Check Status button it displays "Under Construction" 🙏